src/engine/Drawing/Color.ts

```diff
@@ -366,4 +412,10 @@ class HSLColor {
 
       return new Color(r * 255, g * 255, b * 255, this.a);
    }
+
+   public toString(): string {
+      var result = String(this.h.toFixed(0)) + ', ' + String(this.s.toFixed(0)) + ', ' + String(this.l.toFixed(0)) 
```

Comment:
I know this is similar to the rgba one, but I think string templating will help make this clearer:

```ts
let h = this.h.toFixed(0),
    s = this.s.toFixed(0),
    l = this.l.toFixed(0),
    a = this.a.toFixed(0);

return `hsla(${h}, ${s}, ${l}, ${a})`;
```